### PR TITLE
SASS-5731: Create UK dividends, shares and loans from limited companies page

### PR DESCRIPTION
**Pull request by ansahdavid**
### Description
Please include a brief summary of the change you've made and the issue that it fixes.

##### Before creating PR
- [x]  Have you run the tests?
- [x]  Have you run the it tests?
- [ ]  Have you run the journey tests? (where applicable)
- [x]  Have you addressed warnings where appropriate?
- [x]  Have you rebased against the current version of main?
- [x]  Have you checked code coverage isn’t lower than previously?
- [ ]  Have you done the group review in the team?

##### After PRs been raised
- [x]  Have you checked the PR Builder passes?

##### Before Reviewing
- [x]  Have you pulled the branch down?
- [x]  Have you assigned yourself to the PR?
- [x]  Have you moved the task to “in review” on JIRA?
- [ ]  Have you checked to ensure all dependencies are up to date?
- [ ]  Have you checked to ensure its been rebased against the current version of main?

##### Whilst Reviewing
- [x]  Have you run the tests?
- [x]  Have you run the it tests?
- [ ]  Have you run the journey tests?
- [ ]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?
- [x]  Have you checked for individual and agent content matching with the prototype? (where applicable)
- [ ]  Have you checked for welsh transaltion for individual and agent? (where applicable)
- [ ]  Have you checked for errors messages for individual and agent on both english and welsh? (where applicable)
- [x]  Run wave (chrome extension) for accessibility (where applicable)

##### After Reviewing
- [ ]  Have you checked for merge conflicts or any changes in the current main that may affect the current pull request? i.e. does it need another rebase?
- [ ]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [ ]  Have you moved the task to “in pipeline” on Jira?
- [ ]  Demo to Business Analyst

### CODE REVIEW

File: conf/messages.cy [hunk 1]
```diff
@@ -450,4 +450,45 @@ ukInsuranceGains.none = No, I did not make any UK insurance gains
...
+
+ukDividendsSharesAndLoansFromLimitedCompanies.title = UK dividends, shares and loans from limited companies
+ukDividendsSharesAndLoansFromLimitedCompanies.agent.title = UK dividends, shares and loans from limited companies
+ukDividendsSharesAndLoansFromLimitedCompanies.heading = UK dividends, shares and loans from limited companies
+ukDividendsSharesAndLoansFromLimitedCompanies.agent.heading = UK dividends, shares and loans from limited companies
+ukDividendsSharesAndLoansFromLimitedCompanies.detail.understandingDividends.summary = Understanding dividends
+ukDividendsSharesAndLoansFromLimitedCompanies.detail.understandingDividends.p1 = You may get a dividend payment if you own shares in a company. Dividends can be cash or ‘stock’ (shares). You can earn a total of £1,000 from UK and non-UK dividends each year without paying tax. You must tell us about all of your dividend income, even if it’s less than £1,000.
+ukDividendsSharesAndLoansFromLimitedCompanies.detail.taxOnDividends.linkText = Learn more about tax on dividends (opens in a new tab)
```
Comment (ghousedev) [inline conf/messages.cy:461]: (opens in a new tab) text comes as part of the link component, could you please remove that text from the end of this message.

File: conf/messages.cy [hunk 2]
```diff
@@ -450,4 +450,45 @@ ukInsuranceGains.none = No, I did not make any UK insurance gains
...
+ukDividendsSharesAndLoansFromLimitedCompanies.detail.understandingDividendsUnitTrustsInvestmentCompanies.summary = Understanding dividends from unit trusts or investment companies
+ukDividendsSharesAndLoansFromLimitedCompanies.detail.understandingDividendsUnitTrustsInvestmentCompanies.p1 = Investment trusts make money through buying and selling shares or assets in other companies.
+ukDividendsSharesAndLoansFromLimitedCompanies.detail.understandingDividendsUnitTrustsInvestmentCompanies.p2 = Unit trusts make money by buying and selling bonds or shares on the stock market. The fund is split into units which investors buy. A fund manager creates and cancels units when investors join and leave the trust.
+ukDividendsSharesAndLoansFromLimitedCompanies.detail.understandingDividendsUnitTrustsInvestmentCompanies.p3 = Open-ended investment companies are like unit trusts, but create and cancel shares, rather than units, when investors join or leave.
+ukDividendsSharesAndLoansFromLimitedCompanies.detail.understandingDividendsUnitTrustsInvestmentCompanies.p4.linkText = Find out more about authorised investment funds (opens in a new tab)
```
Comment (ghousedev) [inline conf/messages.cy:466]: (opens in a new tab) text comes as part of the link component, could you please remove that text from the end of this message.

File: conf/messages.cy [hunk 3]
```diff
@@ -450,4 +450,45 @@ ukInsuranceGains.none = No, I did not make any UK insurance gains
...
+ukDividendsSharesAndLoansFromLimitedCompanies.closeCompanyLoansWrittenOffReleased = Close company loans written off or released
+ukDividendsSharesAndLoansFromLimitedCompanies.agent.closeCompanyLoansWrittenOffReleased = Close company loans written off or released
+ukDividendsSharesAndLoansFromLimitedCompanies.noUkDividendsSharesOrLoans = No, I did not get any UK dividends, shares or loans
+ukDividendsSharesAndLoansFromLimitedCompanies.agent.noUkDividendsSharesOrLoans = No, I did not get any UK dividends, shares or loans
+ukDividendsSharesAndLoansFromLimitedCompanies.exclusive = Exclusive
+ukDividendsSharesAndLoansFromLimitedCompanies.agent.exclusive = Exclusive
+ukDividendsSharesAndLoansFromLimitedCompanies.checkYourAnswersLabel = Did you get any UK dividends, shares or loans?
+ukDividendsSharesAndLoansFromLimitedCompanies.error.required = Select if you had any UK dividends, shares or loans, or ''No, I did not get any UK dividends, shares, or loans''
+ukDividendsSharesAndLoansFromLimitedCompanies.agent.error.required = Select if your client had any UK dividends, shares or loans, or ''No, my client did not have any UK dividends, shares or loans''
```
Comment (ghousedev) [inline conf/messages.cy:493]: For both of these errors we should use a different type of quotes like the other errors: or ‘No, I did not get any income from UK interest’.

File: conf/messages.cy [hunk 4]
```diff
@@ -450,4 +450,45 @@ ukInsuranceGains.none = No, I did not make any UK insurance gains
...
+ukDividendsSharesAndLoansFromLimitedCompanies.detail.understandingDividendsUnitTrustsInvestmentCompanies.p3 = Open-ended investment companies are like unit trusts, but create and cancel shares, rather than units, when investors join or leave.
+ukDividendsSharesAndLoansFromLimitedCompanies.detail.understandingDividendsUnitTrustsInvestmentCompanies.p4.linkText = Find out more about authorised investment funds (opens in a new tab)
+ukDividendsSharesAndLoansFromLimitedCompanies.detail.understandingFreeRedeemableShares.summary = Understanding free or redeemable shares
+ukDividendsSharesAndLoansFromLimitedCompanies.detail.understandingFreeRedeemableShares.p1 = You can hold both free and redeemable shares.
+ukDividendsSharesAndLoansFromLimitedCompanies.detail.understandingFreeRedeemableShares.p2 = Redeemable shares are ones a company can buy back at an agreed price on a future date.
+ukDividendsSharesAndLoansFromLimitedCompanies.detail.understandingFreeRedeemableShares.p3 = Free additional shares are also known as ''bonus issues of securities''.
```
Comment (ghousedev) [inline conf/messages.cy:470]: I think this should be: ‘bonus issues of securities’ although would be worth double checking this one.

File: conf/messages.cy (hunk 1 above)
Comment (ansahdavid) [reply conf/messages.cy:461]: Yeah I just realised this once you mentioned it, thanks. I'll remove this in my next commit.

File: conf/messages.cy (hunk 2 above)
Comment (ansahdavid) [reply conf/messages.cy:466]: Yeah I just realised this once you mentioned it, thanks. I'll remove this in my next commit.

File: conf/messages.cy (hunk 3 above)
Comment (ansahdavid) [reply conf/messages.cy:493]: Yep thanks for pointing this out. I'll remove this in my next commit.

File: conf/messages.cy (hunk 4 above)
Comment (ansahdavid) [reply conf/messages.cy:470]: Yeah you're most likely to be right, I've asked Rosie to clarify what this should be andI'll make the amemdment in my next commit, Thanks.

Comment (ansahdavid) [reply conf/messages.cy:470]: @ghousedev I've pushed my latest commit to resolve this query, is this what you were expecting?

File: app/controllers/propertypensionsinvestments/UkDividendsSharesAndLoansFromLimitedCompaniesController.scala
Comment (basotia) [inline app/controllers/propertypensionsinvestments/UkDividendsSharesAndLoansFromLimitedCompaniesController.scala:1]: This name is too big, can we please change it to UkDividendsSharesAndLoansController?

**Pull request closed without merging**